README.md

```diff
@@ -18,23 +17,67 @@ Additional code examples can be accessed from the following locations:
+[Enable accuracy monitoring](https://docs.datarobot.com/en/docs/mlops/manage-mlops/setup-accuracy.html) - You can monitor a deployment for accuracy using the Accuracy tab, which lets you analyze the performance of the model deployment over time, using standard statistical measures and exportable visualizations. \
+[MLOps agent](https://docs.datarobot.com/en/docs/mlops/deployment/mlops-agent/index.html) - DataRobot MLOps provides powerful tools for tracking and managing models for prediction. But what if you already have—or need to have—deployments running in your own environment?  \
+[MLOps Agent use cases](https://docs.datarobot.com/en/docs/mlops/deployment/mlops-agent/agent-use.html) - Monitoring use cases below for examples of how to apply the MLOps agent: Reporting metrics, Monitoring a Spark environment, Monitoring using the MLOps CLI
+
+### Portable Prediction Server (PPS) - DataRobot Models in Docker Images Deployed as Containers within Kubernetes
+[Portable Prediction Server](https://docs.datarobot.com/en/docs/mlops/deployment/deploy-pred/portable-pps.html) - The Portable Prediction Server (PPS) is a DataRobot execution environment for DataRobot model packages (.mlpkg files) distributed as a self-contained Docker image. \
+[Portable batch predictions](https://docs.datarobot.com/en/docs/mlops/deployment/deploy-pred/portable-batch-predictions.html) - Portable batch predictions (PBP) let you score large amounts of data on disconnected environments. \
+[Custom model Portable Prediction Server](https://docs.datarobot.com/en/docs/mlops/deployment/deploy-pred/custom-pps.html) - The custom model Portable Prediction Server (PPS) is a solution for deploying a custom model to an external prediction environment. It can be built and run disconnected from main installation environments.  
+
+### Platform Administration
+[Administrator's guide](https://docs.datarobot.com/en/docs/admin/for-admins/index.html) - The DataRobot Administrator's Guide is intended to help administrators manage their DataRobot application. Manage users, Reference. 
 
-This project has adopted the Contributor Covenant for its Code of Conduct.
+### DataRobot API 
+[API Quickstart (documentation](https://docs.datarobot.com/en/docs/api/api-quickstart/index.html) - The DataRobot API provides a programmatic alternative to the web interface for creating and managing DataRobot projects. The API can be used via REST or with DataRobot's Python or R clients in Windows, UNIX, and OS X environments.  \
+[DataRobot University: Python API Starter Quest (Free)](https://university.datarobot.com/path/python-api-starter-quest) - provide the foundation skills for using Python to work with the DataRobot API. The courses are self-paced, so if you prefer instructor-led training, we recommend that you start with API I Python. 
```

Comment:
"we recommend that you start with `API I Python`" -- is this a link another course?